pkg/api/register.go

```diff
@@ -111,6 +111,15 @@ func AddToScheme(scheme *runtime.Scheme) {
 		&unversioned.APIGroup{},
 		&unversioned.APIResourceList{},
 	)
+
+	// Types registered here will be kept unversioned when they are encoded.
+	// It's for backward compatibility with release 1.1. clients.
+	Scheme.AddToKeepUnverioned(
```

Comment:
I'd rather making all of them "" until we have release-v2, than making some groups' endpoints return "" and some return "v1". What do you say?